metadata.rb

```diff
@@ -31,6 +31,8 @@
 depends 'wix'
 depends 'windows-sdk'
 
+gem 'pkg-config'
+
 source_url 'https://github.com/chef-cookbooks/omnibus'
 issues_url 'https://github.com/chef-cookbooks/omnibus/issues'
-chef_version '>= 12.5.1' if respond_to?(:chef_version)
+chef_version '>= 12.8' if respond_to?(:chef_version)
```

Comment:
pretty certain that gem metadata was unusably broken in 12.8 so this line is likely a lie.

also we should be bumping up to at least 12.14 anyway across the board to drop ruby 2.1/2.0 support